src/salted_sha1_fmt_plug.c

```diff
@@ -215,18 +217,16 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 {
 	const int count = *pcount;
 	int index = 0;
+	int inc = 1;
 
-#ifdef _OPENMP
 #ifdef SIMD_COEF_32
-	int inc = NBKEYS;
-#else
-	int inc = 1;
+	inc = NBKEYS;
 #endif
 
+#ifdef _OPENMP
 #pragma omp parallel for
-	for (index=0; index < count; index += inc)
 #endif
-	{
+	for (index=0; index < count; index += inc) {
```

Comment:
Looks like `inc` is a constant, so keeping it in a variable which you first set to 1 and then override to NBKEYS is weird. Even the original code (prior to this patch) wasn't nearly as weird. Also, duplicate `index = 0` above, and missing spaces around the `=`.